pygfx/cameras/_base.py

```diff
@@ -23,17 +25,16 @@ class Camera(WorldObject):
     def __init__(self):
         super().__init__()
 
-        self.matrix_world_inverse = Matrix4()
-        self.projection_matrix = Matrix4()
-        self.projection_matrix_inverse = Matrix4()
+        self.matrix_world_inverse = np.eye(4, dtype=float)
+        self.projection_matrix = np.eye(4, dtype=float)
+        self.projection_matrix_inverse = np.eye(4, dtype=float)
 
     def set_view_size(self, width, height):
         # In logical pixels, called by the renderer to set the viewport size
         pass
 
     def update_matrix_world(self, *args, **kwargs):
-        super().update_matrix_world(*args, **kwargs)
-        self.matrix_world_inverse.get_inverse(self.matrix_world)
+        self.matrix_world_inverse = np.linalg.inv(self.world_transform)
```

Comment:
@FirefoxMetzger This is no longer called, so the inverse is never calculated.

Reply:
Where do we use the inverse world matrix? Just for cameras and lights?

Reply:
Only for the camera to get what I think we called the view transform. See the snipped that Korijn shared above, particularly this line:
```py
        stdinfo_data["cam_transform"].flat = camera.matrix_world_inverse.elements 
```